templates/webpack-basic-typescript/package.json

```diff
@@ -1,31 +1,33 @@
 {
   "name": "self.id-template-webpack-basic-typescript",
-  "version": "0.1.0",
+  "version": "0.0.0",
```

Comment:
Why does this change from `0.1.0`to `0.0.0`?

Reply:
Just for consistency with the other templates, the version doesn't matter here, we just need to set one to avoid having npm warn about it